src/datta_lab_to_nwb/markowitz_gillis_nature_2023_keypoint/irvideointerface.py

```diff
@@ -0,0 +1,78 @@
+"""Base class for converting raw video data."""
+from pynwb import NWBFile
+import numpy as np
+from pathlib import Path
+from neuroconv.datainterfaces import VideoInterface
+from ..markowitz_gillis_nature_2023.basedattainterface import BaseDattaInterface
+
+
+class IRVideoInterface(BaseDattaInterface):
+    """IR video interface for markowitz_gillis_nature_2023 conversion of keypoint data."""
+
+    def __init__(
+        self,
+        data_path: str,
+        session_uuid: str,
+        session_id: str,
+        session_metadata_path: str,
+        subject_metadata_path: str,
+    ):
+        super().__init__(
+            data_path=Path(data_path),
+            session_uuid=session_uuid,
+            session_id=session_id,
+            session_metadata_path=session_metadata_path,
+            subject_metadata_path=subject_metadata_path,
+        )
+
+    def add_to_nwbfile(self, nwbfile: NWBFile, metadata: dict) -> None:
+        SAMPLING_RATE = 30
```

Comment:
All capitals is usually a convention for environment/global variables but this is defined only in the local scope - was there a reasoning here?

Reply:
I typically follow the convention that named constants should be in all caps regardless of the scope (see [RealPython](https://realpython.com/python-constants/#:~:text=Because%20Python%20constants%20are%20just,uppercase%20letters%20(%20A%20%E2%80%93%20Z%20))) hence the usage here.

That being said, since SAMPLING_RATE is used by both the keypointinterface and the irvideointerface, I should really define it in the metadata .yaml file.  Will fix.